src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -78,12 +73,13 @@ export const NavBar: React.FC = track(
     user,
     "User Conversations View"
   )
+  // ToDo: replace check for conversationsEnabled with check for user when ready to launch
+  const showNotificationCount = !showMobileMenu && conversationsEnabled
```

Comment:
Minor suggestion to check if the user is logged in additionally so when we are launching we can simply remove the `conversationsEnabled` check.

```suggestion
  const showNotificationCount = isLoggedIn && !showMobileMenu && conversationsEnabled
```